components/Premium/Premium.tsx

```diff
@@ -122,6 +122,8 @@ function Premium() {
                     >
                         <span>Your premium ends: {moment(hasPremiumUntil).fromNow()}</span>
                     </OverlayTrigger>
+
+                    <p style={{ color: 'yellow' , marginTop: 15 }}>This is not a recurring subscription. We won't automatically charge you after your premium time runs out!</p>
```

Comment:
Put the text under the "Premium Plans" headline

Reply:
As there is no "Premium Plans" anymore, I would put this information as a sub-title under the 
"Buy premium for a certain duration with your CoflCoins. The premium activate shortly after your purchase."-Title